contrail_api_cli/commands/batch.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import shlex
+import sys
+import fileinput
+from six import text_type
+
+from keystoneauth1.exceptions.http import HttpError, HTTPClientError
+
+from ..command import Command, Arg, Option
+from ..manager import CommandManager
+from ..context import Context
+from ..schema import SchemaError
+from ..exceptions import CommandError, CommandNotFound, NotFound, Exists
+from ..utils import printo
+
+
+class Batch(Command):
+    """ Execute commands from file(s) or from stdin
```

Comment:
Can you explain the format of the file in the docstring ?

Reply:
This is displayed when running `contrail-api-cli man batch`

Reply:
Yes, of course. Sorry, the original docstring was just a basic short description. I did not focus on that. Now I enhanced it.